packages/react-core/src/components/FormSelect/FormSelectOption.tsx

```diff
@@ -9,13 +9,17 @@ export interface FormSelectOptionProps extends Omit<React.HTMLProps<HTMLOptionEl
   label: string;
   /** flag indicating if the option is disabled */
   isDisabled?: boolean;
+  /* flag indicating if option will have placeholder styling applied when selected*/
```

Comment:
```suggestion
  /* flag indicating if option will have placeholder styling applied when selected */
```
This missing space is causing this text not to show up for prop description on the site.